### DISCUSSION TITLE
New Cross Platform Library Doesn't compile - Could not copy the file ...  & Program does not contain a static Main method suitable for an entry point [wasm] (Uno.Templates::5.1.15 installed)

### DISCUSSION THREAD

**Discussion by ToddGlodek** [unoplatform/uno#15347] [General]
Attempted to create a new Cross Platform Library (revisiting the topic #12188).  Used the new project wizard to create a Cross Platform Library with Uno.Templates::5.1.15 installed.   Immediately after creating new project ( made no modifications ) attempted a build of the newly created project.

OBSERVED:  Project compile failed with several errors:

1. Could not copy the file in the main project 
2. Program does not contain a static Main method suitable for an entry point in the WASM project

![image](https://github.com/unoplatform/uno/assets/19764509/063af806-7213-4008-8364-4bd95f5ac606)

EXPECTED:  A newly created project should compile if it has not had any modifications.

**Comment by jeromelaban** [comment]
Thanks for the report. The wizard is not supposed to be able to create cross platform libraries. Can show the command line you used to create this library?

**Reply by ToddGlodek** [reply]
I just copied/pasted the sample expression from [dotnet new templates for Uno Platform](https://platform.uno/docs/articles/get-started-dotnet-new.html?tabs=windows)

`dotnet new unolib-crossruntime -o MyCrossRuntimeLibrary`

I double-checked the using uno-check.  The only thing I don't have installed is the Hyper-V accelerator - but that should not affect the compile process.

![image](https://github.com/unoplatform/uno/assets/19764509/6eb8c453-63c2-4db1-b0e6-94fa0cdfaa9d)

**Reply by jeromelaban** [reply]
Thanks for the report. Let's track in https://github.com/unoplatform/uno.templates/issues/553. In the meantime, you can use the uno.templates package from Uno 5.0.
